tests/lib/rules/void-dom-elements-no-children.js

```diff
@@ -54,6 +54,10 @@ ruleTester.run('void-dom-elements-no-children', rule, {
     }, {
       code: 'React.createElement("img");'
     }, {
+      code: 'React.createElement();'
```

Comment:
I feel like this shouldn't be valid; altho maybe this rule shouldn't be handling it.